features/verblijfplaats.feature

```diff
@@ -0,0 +1,140 @@
+      | datumInschrijvingInGemeente.type  | Datum                    |
+      | datumInschrijvingInGemeente.datum | 2002-07-01               |
+      | land.code                         | <land>                   |
+      | adresregel1                       | <adresregel1>            |
+      | adresregel2                       | <adresregel2>            |
+      | datumVan.type                     | Datum                    |
+      | datumVan.datum                    | 2022-01-28               |
+
+      Voorbeelden:
+      | land | land omschrijving            | adresregel1                 | adresregel2                |
+      | 6014 | Verenigde Staten van Amerika | 1600 Pennsylvania Avenue NW | Washington, DC 20500       |
+      | 9999 | Internationaal gebied        |                             | Amundsen-Scott, Antarctica |
+
+  Rule: Een verblijfplaats is een binnenlands adres wanneer straat (11.10) en/of huisnummer (11.20) een waarde heeft
+    - veld 'type' krijgt de waarde 'Adres' 
+
+    Abstract Scenario: verblijfplaats binnenlands adres
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                 | waarde        |
+      | gemeenteVanInschrijving (09.10)      | 0518          |
+      | gemeenteVanInschrijving.omschrijving | 's-Gravenhage |
+      | datumInschrijvingInGemeente (09.20)  | 20020701      |
+      | functieAdres (10.10)                 | B             |
+      | datumAanvangAdreshouding (10.30)     | 20220128      |
+      | straatnaam (11.10)                   | Spui          |
+      | huisnummer (11.20)                   | 70            |
+      | postcode (11.60)                     | 2511BT        |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                             |
+      | type                | RaadpleegMetBurgerservicenummer    |
+      | burgerservicenummer | 555550001                          |
+      | fields              | burgerservicenummer,verblijfplaats |
+      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'verblijfplaats' gegevens
+      | naam                                 | waarde                 |
+      | type                                 | Adres                  |
+      | gemeenteVanInschrijving.code         | 0518                   |
+      | gemeenteVanInschrijving.omschrijving | 's-Gravenhage          |
+      | datumInschrijvingInGemeente.type     | Datum                  |
+      | datumInschrijvingInGemeente.datum    | 2002-07-01             |
+      | functieAdres.code                    | B                      |
+      | straat                               | Spui                   |
+      | korteNaam                            | Spui                   |
```

Comment:
Hier is korteNaam als property opgenomen, maar er is geen specificatie hoe deze wordt bepaald. Idem voor adresregel1 en adresregel2